docs/topics/auth/customizing.txt

```diff
@@ -407,7 +407,13 @@ use as your User model.
    have a ForeignKey to each other and seeing how ``makemigrations`` resolves that
    circular dependency if you want to see how it's usually done)
 
+.. warning::
 
+   Reusable apps don't use this feature. Remember: One project has N app. If 
```

Comment:
Yes, this was a typo. Thank you for looking at my patch. Can you correct it, or should I do it?